packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -209,11 +210,13 @@ export class CeramicCliUtils {
       if (syncOverride) {
         config.node.syncOverride = syncOverride
       }
+      if (disableComposedb) {
+        config.indexing.disableComposedb = true
+      }
 
-      config.indexing.composedbEnabled = fromBooleanInput(enableComposedb, true)
-      if (process.env.CERAMIC_ENABLE_COMPOSE_DB) {
-        config.indexing.composedbEnabled = fromBooleanInput(
-          process.env.CERAMIC_ENABLE_COMPOSE_DB,
+      if (process.env.CERAMIC_DISABLE_COMPOSE_DB) {
+        config.indexing.disableComposedb = fromBooleanInput(
```

Comment:
if you make the change above you can just set to true here (which is how we handle other boolean flags, so would be consistent).  Or you can leave it like this but make the default value be the current value in the config object

Reply:
yeah, I'll do how the others are handled. that way I remove the dependency to the helper function as well.